source/glossary.rst

```diff
@@ -0,0 +1,418 @@
+
+   Administrative closing
+      Modification of a ticket status to closed (manually or automatically).
+
+   Alert threshold
+      Minimal value from which an alert is triggered.
+
+   Automatic action
+      Process for regular actions triggering, either internal to GLPI or external (Unix cron, Windows planned tasks...).
+
+   Automatic closing
+      GLPI internal process that closes unresolved tickets after a given time.
+
+   Bookmark
+      Stored link to a GLPI page that allows to access the page quickly.
+
+   Budget
+      Accounting operation that groups all provisional expenses and receipts.
+
+   Cartridge
+      Consumable used by a printer and specifically managed in GLPI.
+
+   Case
+      Physical element that contains a computer's components (motherboard, processor, disk...).
+
+   SSO
+      Unique authentication system (:abbr:`SSO (Single Sign On)`).
+
+   Accreditation
+      Authorization to perform a set of actions on several entities, given by rights attribution process.
+
+      .. ??? Autorisation résultante de l'attribution de droits sur un périmètre d'action (une ou plusieurs entités).
+
+   Closed (ticket)
+      Status of a ticket whose solution has been approved by the ticket issuer or that has been close automatically.
+
+   Connection (network)
+      Link between two network ports.
+
+   Registered user
+      Person having an accreditation allowing to connect to GLPI.
+
+   Contact
+      Referent person linked to a supplier.
+
+   Contract
+      Financial document linked to assets and to suppliers
+
+   Controler
+      IT component
+
+      .. ??? Composant informatique.
+
+   Criteria
+      Element or pattern used by search engines to select an information or used by rules engines and dictionaries to perform actions.
+
+   Depreciation type
+      Characteristic of depreciation which balance can be linear or declining
+
+   Default profile
+      Profile attributed by the application in the absence of particular settings (rules, manual attribution...).
+
+   Dictionary
+      Set of rules that allow to modify GLPI data.
+
+   Direct connection
+      Physical link between a computer and another asset.
+
+   Active Directory service
+      Directory service compatible with :term:`LDAP directory`, provided by Microsoft. Also known as `AD`.
+
+   LDAP directory
+      Directory service using the :abbr:`LDAP protocol (Lightweight Directory Access Protocol)`. Allows in particular to store user accounts.
+
+   Document
+      Element that defines a link and/or store a file and that can be associated to other objects of GLPI.
+
+   Mail domain
+      Part of a mail adress that is after character `@`.
+
+      .. ??? Domaine de messagerie
+
+   Domain
+      Grouping of assets connected to a network.
+
+      .. ??. Regroupement de matériels connectés au réseau.
+
+   Title
+      Customizable element of a a drop-down list
```

Comment:
```suggestion
      Customizable element of a drop-down list
```
I don't think this is called Title anywhere in GLPI under the default language.

Reply:
In French it is "intitulé", which can be translated as "Entitled", "Named", "Designation". Don't know how it is called in GLPI UI.

Reply:
It is just called "Dropdowns" in the menu for en_GB and en_US.

Reply:
Maybe something like:
```
   Dropdown
      Customizable dropdowns list used in GLPI and plugins.
```

Reply:
Agree and changed.